todolist copy 3/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+		<title>To do list</title>
+		<meta name="viewport" content="width=device-width,initial-scale=1">
+		<link rel="stylesheet" href="style.css">
+   </head>
+   <body>
+      <div class="container">
+         <div class="menu">
+            <h1>TaskDo</h1>
+         </div>
+         <div class="add-new-task">
+			<input type="text "id="taskInput" placeholder="Add New Task" required>
+            <button class="button-click" id="addTask">+</button>
+            Add New Task
+         </div>
+		 <button id="clearAll">Clear All</button>
+         <ul class="tasklist" id="tasklist">
```

Comment:
1) task list is two words 
2) we need to decide on the naming convention, is it gonna be camel case, snake case, kebab case, etc, now there is no consistency
"clearAll", "tasklist" => "clearAll", "taskList" or "clear-all", "task-list" or "clear_all", "task_list"
For example, the kebab case is mostly used in BEM 

https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841 (if you reached your medium limit, just open it in incognito window)